toolchain/defs.bzl

```diff
@@ -147,12 +147,8 @@ def _zig_repository_impl(repository_ctx):
 
     cache_prefix = repository_ctx.os.environ.get("HERMETIC_CC_TOOLCHAIN_CACHE_PREFIX", "")
     if cache_prefix == "":
-        if os == "windows":
-            cache_prefix = "C:\\\\Temp\\\\zig-cache"
-        elif os == "macos":
-            cache_prefix = "/var/tmp/zig-cache"
-        elif os == "linux":
-            cache_prefix = "/tmp/zig-cache"
+        if os == "windows" or os == "macos" or os == "linux":
+            cache_prefix = "zig-cache"
```

Comment:
Can we put it in either `bazel-zig-cache` or `bazel-out/zig-cache`? Most bazel project have either `/bazel-*` or `/bazel-out` in their `.gitignore`. They don't need to ignore additional directory if we reuse the existing directories.